packages/runner/injection/patches/cookies.ts

```diff
@@ -0,0 +1,133 @@
+import { AutomationCookie } from '@packages/server/lib/automation/cookies'
```

Comment:
Could we break automationCookie out into its own file? This import is pulling in an awful lot just to grab the interface.

```js
import _ from 'lodash'
import Debug from 'debug'
import extension from '@packages/extension'
import { isHostOnlyCookie } from '../browsers/cdp_automation'
```

Reply:
Changed it to use `import type`, so it shouldn't pull anything in to the bundle. Addressed in https://github.com/cypress-io/cypress/pull/23643/commits/7b91ba5cde2a1fb690c3e08642e7909c5fd4522a